Codeinwp/neve
# Print styles | menu scroll | Shift-click HFG | Customizer panel overflow

### Summary


### Will affect visual aspect of the product

NO

### Test instructions


- Check if when hitting Ctrl+P everything looks ok in print mode
- Open menu using responsive mode, the page should be scrollable
- Shift click header/footer components, the corresponding customizer section should get focused
- Fixed an issue where the customizer content for some panel had too much height, allowing scrolling even when it wasn't necessary


Closes #552, closes #583. closes #641.
